.github/CHANGELOG.md

```diff
@@ -126,6 +126,14 @@
 1. [Sound] Improved nose wheel liftoff sound - @hotshotp (Boris)
 1. [Sound] Added new APU start contactor sounds - @hotshotp (Boris)
 1. [Sound] Added new realistic LEAP-1A engine sounds - @hotshotp (Boris)
+1. [Lights] Take-Off Lights | Intensity has increased | Range has decreased | Rotation has changed - @Snapmatics (Harry)
+1. [Lights] Landing Lights | Intensity has decreased | Range has decreased | Not Final Look  - @Snapmatics (Harry)
+1. [Lights] Strobe Lights | Intensity has increased | Range has decreased | Rotation has changed | There is no bleeding into the cockpit anymore - @Snapmatics (Harry)
+1. [Lights] Beacon Top Lights | They now reflect on the wings - @Snapmatics (Harry)
+1. [Lights] Beacon Belly Lights | They now reflect on the engines - @Snapmatics (Harry)
+1. [Lights] Flood Lights - @Snapmatics (Harry)
+1. [Lights] Reading Lights - @Snapmatics (Harry)
+1. [Lights] PFD/ND/EICAS/MCDU Lights Updated - @Snapmatics (Harry)
```

Comment:
Couldn't you just summarise this?

Reply:
Really no. Some people want to see the exact changes without reading a whole paragraph. I will ask Iceman for that.

Reply:
I would recommend summarising this tbf. No need for 8 lines of changes for the same PR. 2 at most would do the job

Reply:
Done!